lib/chef/knife/bootstrap.rb

```diff
@@ -146,6 +146,10 @@ class Bootstrap < Knife
         description: "The version of Chef to install",
         proc: lambda { |v| Chef::Config[:knife][:bootstrap_version] = v }
 
+      option :prerelease,
```

Comment:
any reason not to just expose a channel flag. That's a term people would be familiar with and we'd avoid any sort of magic by just passing in the channel name. That would also make it future proof if we added more channels in the future.

Reply:
Agree with @tas50, let's add a "channel" option. Then instead of adding `-p` to the installer_version string (which is just passed to install.sh),  we would pass `-c CHANNEL`.

The prerelease option could warn it is deprecated and set the channel to "current" which is functionally our prerelease builds now.

Reply:
Cool, I'd had the same thought and figured few changes are better at this point.

Reply:
it's a small change that's overdue. I think it will also let us install from unstable which will let us get the license acceptance builds that we'll start making soon that we're going to intentionally keep from hitting current for a couple days.